modules/nw-egress-ips-about.adoc

```diff
@@ -80,6 +76,13 @@ The annotation value is an array with a single object with fields that provide t
 * `ifaddr`: Specifies the subnet mask for one or both IP address families.
 * `capacity`: Specifies the IP address capacity for the node. On AWS, the IP address capacity is provided per IP address family. On Azure and GCP, the IP address capacity includes both IPv4 and IPv6 addresses.
 
+Automatic attachment and detachment of egress IP addresses for traffic between nodes are available. This allows for traffic from many pods in namespaces to have a consistent source IP address to locations outside of the cluster. This also supports OpenShift SDN and OVN-Kubernetes, which is the default networking plug-in in Red Hat OpenShift Networking in {product-title} {product-version}.
+
+[NOTE]
+====
+The {rh-openstack} egress IP address feature creates a neutron reservation port called `egressip-<IP address>`. Using the same {rh-openstack} user as the one used for the {product-title} cluster installation, you can assign a floating IP address to this reservation port to have a predictable SNAT address for egress traffic. When an egress IP address on an {rh-openstack} network is moved from one node to another, because of a node failover, for example, the neutron reservation port is removed and recreated. This means that the floating IP association is lost and you need to manually reassign the floating IP address to the new reservation port.
```

Comment:
Should _neutron_ be capitalized?

Reply:
Thanks @mburke5678 I made that capitalization change.